service/local-dev/README.md

```diff
@@ -1,33 +1,41 @@
 This directory contains scripts for running continuous development builds. This
 is not necessary for deployment, but it can be helpful for developing faster.
+It clones the terra-helmfile git repo to get the chart and values files for deploying to
+your personal environment.
 
-The provided setup script clones the terra-helm and terra-helmfile git repos,
-and templates in the desired Terra environment/k8s namespace to target.
-If you need to pull changes to either terra-helm or terra-helmfile, rerun this script.
-
-You may need to use gcloud to provide GCR
- credentials with `gcloud auth configure-docker`. Finally, run local-run.sh with
-  your target environment as the first argument:
-
+Invoke the script as follows:
 ```
-./setup_local_env.sh <environment>
+./setup_local_env.sh <environment> [ssh|http] [<terra-helmfile ref>] 
 ```
 
-Optionally, you can provide a branch of [terra-helm](https://github.com/broadinstitute/terra-helm) and/or [terra-helmfile](https://github.com/broadinstitute/terra-helmfile) to use if you are testing PRs to either. They default to master. A git protocol (`ssh` or `http`) can optionally be specified as a fourth argument to the script.
-```
-./setup_local_env.sh <environment> [<terra-helm ref>] [<terra-helmfile ref>] [ssh|http]
+Where `<environment>` is the name of your personal environment.
```

Comment:
just confused about terminology here. what is my personal environment?

Reply:
I provided a bit more context in the README.